fix: example to show cron binding usage (#1869)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-bindings/cron.md

* `@every 15s` where `s` is seconds, `m` minutes, and `h` hours
* `@daily` or `@hourly` which runs at that period from the time the binding is initialized

## Listen to the cron binding

> After setting up the cron binding, all you need to do is listen on an endpoint that matches the name of your component. Assume the <NAME> is `scheduled`. Note that this will be made as a HTTP `POST` request. The below example shows how a simple Node.js Express application can receive calls on the `/scheduled` endpoint and write a message to the console.

Reviewer: @swuecho, can you please use a different wrapper for "<NAME>". Our docs will interpret this as HTML (as does GitHub).

Reviewer: ```suggestion
After setting up the cron binding, all you need to do is listen on an endpoint that matches the name of your component. Assume the <NAME> is `scheduled`. This will be made as a HTTP `POST` request. The below example shows how a simple Node.js Express application can receive calls on the `/scheduled` endpoint and write a message to the console.
```

> After setting up the cron binding, all we need to do is listen on an endpoint that matches the name of our component. let's assume the <NAME> is `scheduled`. Note that this will be made as a HTTP `POST` request. The example below shows how a simple Node.js Express application can receive calls on the `/scheduled` endpoint and write a message to the console.

Reviewer: Hi @swuecho,

Thank you for this contribution. Some minor suggestions based on docs style and tone guidance: https://docs.dapr.io/contributing/contributing-docs/#style-and-tone

 ```suggestion
After setting up the cron binding, all you need to do is listen on an endpoint that matches the name of your component. Assume the <NAME> is `scheduled`. Note that this will be made as a HTTP `POST` request. The below example shows how a simple Node.js Express application can receive calls on the `/scheduled` endpoint and write a message to the console.
```

  
```js
app.post('/scheduled', async function(req, res){
    console.log("scheduled endpoint called", req.body)
    res.status(200).send()
});
```
  

> If we run this, we'll see that the /scheduled endpoint is called every five minutes by the Dapr sidecar.

Reviewer: ```suggestion
When running this code, note that the `/scheduled` endpoint is called every five minutes by the Dapr sidecar.
```

- [How-To: Trigger application with input binding]({{< ref howto-triggers.md >}})
- [How-To: Use bindings to interface with external resources]({{< ref howto-bindings.md >}})
- [Bindings API reference]({{< ref bindings_api.md >}})

> - [Scheduled tasks with the Cron binding in Dapr](https://markheath.net/post/dapr-cron-binding)

Reviewer: @swuecho can you please remove this external link?
```suggestion
```
